### DISCUSSION TITLE
Why neovide is not working?

### DISCUSSION THREAD

**Discussion by tonmoy998** [LazyVim/LazyVim#741] [Q&A] [answered]
Sometimes neovide automatically stops

**Comment by folke** [comment, marked as answer]
Caused by bugs in Neovim/Neovide triggered by Noice. Best to disable noice if you want to use neovide

**Reply by Jomik** [reply]
Piggy-backing off this. I cannot seem to disable it based on the `neovide` variable?
I try to use the below, but it always loads?
```lua
return  { "folke/noice.nvim", cond = not vim.g.neovide }
```

**Reply by Jomik** [reply]
Right, I need to do something like this.. Can I make that prettier..?
```lua
return    { "folke/noice.nvim", cond = not (vim.g.neovide or false) }
```

**Reply by tonmoy998** [reply]
does it turn off while using neovide?

**Comment by tonmoy998** [comment]
Thank you for your information

**Comment by Dronakurl** [comment]
I am currently using neovim with nightly version (NVIM v0.11.0-dev-348+g3c53e8f78) and neovide 0.13.1.
Noice is enabled, no random crashes so far.

the only issue was that the bottom line for search had "background=foreground" and characters didn't show. Therefore I set "opts.presets.bottom_search = false".
